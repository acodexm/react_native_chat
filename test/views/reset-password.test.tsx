import React from 'react';
import ResetPassword from '@views/auth/reset-password/reset-password';
import MockedComponent from '../support/mocked-component';
import { render } from '@testing-library/react-native';

it('renders correctly', () => {
  const tree = render(<MockedComponent Component={ResetPassword} />).toJSON();
  expect(tree).toMatchSnapshot();
});
