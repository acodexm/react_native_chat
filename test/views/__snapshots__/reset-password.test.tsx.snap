// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#ffffff",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "paddingBottom": 15,
        "paddingLeft": 15,
        "paddingRight": 15,
        "paddingTop": 15,
      },
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
        },
      ]
    }
  >
    <View
      style={
        Array [
          Object {
            "backgroundColor": "#ffffff",
            "flexBasis": 0,
            "flexGrow": 1,
            "flexShrink": 1,
            "paddingBottom": 15,
            "paddingLeft": 15,
            "paddingRight": 15,
            "paddingTop": 15,
          },
        ]
      }
    >
      <TextInput
        allowFontScaling={true}
        onBlur={[Function]}
        onChangeText={[Function]}
        placeholder="email.placeholder"
        rejectResponderTermination={true}
        style={
          Array [
            Object {
              "borderColor": "#4e4e4e",
              "borderWidth": 1,
              "marginBottom": 5,
              "paddingBottom": 12,
              "paddingLeft": 12,
              "paddingRight": 12,
              "paddingTop": 12,
            },
          ]
        }
        testID="email"
        underlineColorAndroid="transparent"
        value=""
      />
      <View
        accessibilityRole="button"
        accessibilityState={Object {}}
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
        testID="submit"
      >
        <View
          style={
            Array [
              Object {},
            ]
          }
        >
          <Text
            disabled={false}
            style={
              Array [
                Object {
                  "color": "#007AFF",
                  "fontSize": 18,
                  "margin": 8,
                  "textAlign": "center",
                },
                Object {
                  "color": "#3740FE",
                },
              ]
            }
          >
            button.reset
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;
